swan-lake/development-tutorials/deployment-guide/virtual-machine-deployment.md

```diff
@@ -1,12 +1,22 @@
 ---
 layout: ballerina-cloud-left-nav-pages-swanlake
 title: Virtual machine deployment
-description: If you prefer to execute Ballerina services directly on virtual machines (VMs) or bare metal servers without using containerization, you can follow these steps to set up and run your services in a production environment, 
+description: This guide explains how to deploy Ballerina services on virtual machines (VMs), offering a traditional server-based approach without containerization. It covers two deployment strategies, non-centralized and centralized.
 keywords: ballerina, programming language, services, cloud, kubernetes, docker
 active: ballerina-deployment-guideLines
-intro: If you prefer to execute Ballerina services directly on virtual machines (VMs) or bare metal servers without using containerization, you can follow these steps to set up and run your services in a production environment, 
+intro: This guide explains how to deploy Ballerina services on virtual machines (VMs), offering a traditional server-based approach without containerization. It covers two deployment strategies, non-centralized and centralized.
 ---
 
+## Non-centralized deployment
```

Comment:
Should we use the term decentralized?

Reply:
Fixed via [4c1f3f0](https://github.com/ballerina-platform/ballerina-dev-website/pull/9800/commits/4c1f3f0911c65709821f49e7f6ae08d244becf46)